### PR TITLE
Refactoring of Main Components

### PR DESCRIPTION
**Pull request by nicoquiroga90**
Some redundant code was removed, and the components for team creation and submission were simplified. Additionally, the database tables were modified according to the new schema, and the code was adapted to the new column names.

### CODE REVIEW

File: src/component/CreateTeam.jsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
-const CreateTeam = ({ setTeamsDatabase, setTeamCode }) => {
+import '../index.css';
+
+const CreateTeam = ({ setTeamsDatabase }) => {
   const handleCreateTeam = async () => {
     try {
       const randomCode = Math.random().toString(36).substring(2, 8).toUpperCase();
 
-      const title = prompt("Enter new team name:");
+      const newName = prompt("Enter new team name:");
```
Comment (aina21) [inline src/component/CreateTeam.jsx]: How about `newTeamName` or `teamName`?

File: src/component/TeamForm.jsx
Comment (aina21) [inline src/component/TeamForm.jsx:1]: Was removing this file part of a cleanup effort, or is there another reason behind it? Just trying to catch up. :)

File: src/component/CreateTeam.jsx (hunk 1 above)
Comment (nicoquiroga90) [reply src/component/CreateTeam.jsx]: newTeamName sound better :)

File: src/component/TeamForm.jsx
Comment (nicoquiroga90) [reply src/component/TeamForm.jsx:1]: Yes, it was doing almost the same as CreateTeam

**Comment by nicoquiroga90** [comment]
Closed to evoid merge conflict, the refactoring will keep in the branch "Authentication-NicoQ"

**Pull request closed without merging**